pkgs/top-level/python-packages.nix

```diff
@@ -21784,6 +21784,8 @@ EOF
   pysigset = callPackage ../development/python-modules/pysigset { };
 
   us = callPackage ../development/python-modules/us { };
+
+  diceware = callPackage ../tools/security/diceware { };
```

Comment:
This is an application and should therefore be called directly from `all-packages.nix`, and not via `python-packages.nix`.